lib/build.js

```diff
@@ -8,23 +8,41 @@ const pify = require('pify');
 const writeFile = pify(fs.writeFile);
 
 module.exports = context => {
+	const version = parseInt(context.version.split(/\./g)[0], 10);
+
 	const dockerfile = [
 		`FROM node:${context.version}`,
 		'WORKDIR /usr/src/app',
 		'ARG NODE_ENV',
 		'ENV NODE_ENV $NODE_ENV',
-		'COPY package.json .',
-		'RUN npm install',
-		'COPY . .',
+		'COPY . .'
```

Comment:
`COPY . .` should always be last in the Dockerfile here, could you add it below where you add `npm start`?

Reply:
I've moved it at the top because `npm ci` requires the file `package-lock.json` or `npm-shrinkwrap.json`. And I also don't know why it need to be at the bottom.

Reply:
You could copy `package-lock.json` individually then: `COPY package-lock.json ./`. It has to be at the bottom to ensure a change in any file doesn't trigger re-installation of npm dependencies.